server/Dockerfile

```diff
@@ -19,6 +19,7 @@ COPY server/core/ server/core/
 COPY server/rest/ server/rest/
 COPY server/log/ server/log/
 COPY server/persistence/ server/persistence/
+COPY server/workspacesutil server/workspacesutil
```

Comment:
For a future PR: copy the entire `server/` directory, and exclude files with a `.dockerignore` file.